guides/v2.3/ui_comp_guide/components/ui-wysiwyg.md

```diff
@@ -69,6 +69,19 @@ Wysiwyg-specific options:
   </tr>
 </table>
 
+## Events
+
```

Comment:
Maybe add a short description of what Events are and how a developer may use them.